README.md

```diff
@@ -464,6 +465,10 @@ The structure of the JSON results is as follows:
                                 # (structure same as 'core_cycle_counts')
     'mperf_counts': {...}       # Per-core MPERF deltas
                                 # (structure same as 'core_cycle_counts')
+    'pexec_flags': {...}        # A flag for each process execution:
```

Comment:
Do we need to document `O` too?

Reply:
`O` cannot make it into the results file because an outstanding pexec doesn't have a result yet. Same applies for `S`.